### ISSUE TITLE
Blacked blank window when lauch Bodymovin

### ISSUE THREAD

**Issue by Luan-pham109** [bodymovin/bodymovin-extension#30]
Hello!
Have a problem couldn’t find a solution.
When I’m launching Bodymovin plugin, only blank black window appears.
What it may be and how can I fix it?
After Effects cc 2018
![image](https://user-images.githubusercontent.com/57979545/69213961-36c47280-0b98-11ea-9855-a6df18033434.png)

Many thanks!

**Comment by bodymovin** [comment]
Hi, is there any chance you have something running on port 3000?

**Comment by Luan-pham109** [comment]
Sorry, I dont really know what is "port 3000". Could you give me some more detail. Many thanks

**Comment by bodymovin** [comment]
Can you check if you have anything else running that you can close?

**Comment by doup** [comment]
In my case I'm getting a grey screen in After FX CC 2017.2. After following the debugging instructions and opening the `React Panel` from the [CEFClient](https://github.com/Adobe-CEP/CEP-Resources/tree/master/CEP_8.x), I only see the following error:

- `main.29046264.js:33827 Uncaught TypeError: Cannot read property 'require' of undefined`
    - Which is: `return window.cep_node.require('path');`

So, basically `cep_node` does not exist. Should I install any other extension before installing Bodymovin?

**Comment by doup** [comment]
Btw, I've tried to inspect `cep` and `cep_node` values on the console but when hitting enter it doesn't do anything, it adds a new line but doesn't execute the code. Same happens if I try to inspect `window` or run `console.log(1);`. It's quite crappy this CEFClient. ^_^U

**Comment by bodymovin** [comment]
@doup you shouldn't need to install or load any other dependencies. Adobe CEP panels come with their own Node instance.
It's very strange that those apis are not available when loading the panel.

**Comment by doup** [comment]
The issue happens when using an old After FX version which uses a different CEP API version.

**Comment by jasecoop** [comment]
I have the same issue here, any updates?

**Comment by wahyufaturrizky** [comment]
Me too, I have the same issue here, any updates?
